src/routes/Home.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Characters from '../Components/Characters';
+import Navbar from '../Components/Navbar';
+import GetDisney from '../Redux/Disney/GetDisney';
+
+import Container from './Styles/Home.style';
+
+function Home() {
+  const dispatch = useDispatch();
+  const characters = useSelector((state) => state.Disney.characters);
+  useEffect(() => {
+    dispatch(GetDisney());
+  }, []);
+  return (
+    <div>
+      <Navbar heading="Disney Characters" />
+      <Container>
+        {characters.map((character) => (
+          <Characters
```

Comment:
- [ ] Good job so far. However, kindly implement a Filter stateless component to filter data retrieved from the API and implement a filter or Search feature so that users can search for a character without having to scroll all the way to find the character. You can look for how to implement a search feature in React on YouTube.

Reply:
Thank you. I implemented the search feature.